build/dockerfiles/KServe/README.md

```diff
@@ -0,0 +1,56 @@
+# Kitops ClusterStorageContainer image for KServe
+
+The Dockerfile in this directory is used to build an image that can run as a ClusterStorageContainer for [KServe](https://kserve.github.io/website/master/).
+
+## Building
+To build the image, `docker` or `podman` is required. From the root of this repository, run
+```bash
```

Comment:
Let's use docker as part of the example. Podman may confuse people. I have already checked with docker, command works as expected

Reply:
Mentioned above -- how do we usually document a docker build like this 🤔